src/util/lp/core_solver_pretty_printer_def.h

```diff
@@ -38,7 +38,7 @@ core_solver_pretty_printer<T, X>::core_solver_pretty_printer(lp_core_solver_base
     m_rs(ncols(), zero_of_type<X>()),
     m_w_buff(core_solver.m_w),
     m_ed_buff(core_solver.m_ed) {
-    m_low_bounds_title = "low";
+    m_lower_bounds_title = "low";
     m_upp_bounds_title = "upp";
```

Comment:
low was changed to lower, but upp was not changed.

Reply:
The short names snug nicely into a column